src/transformer/transformer.py

```diff
@@ -61,29 +61,19 @@ def __transform_json(self, json: bytes, transformation: str) -> None:
 
     def __transform_xml(self, xml: bytes, transformation: str) -> str:
         xslt_path = self.__get_path_to_xslt(transformation)
-        saxon_path = self.__get_path_to_saxon()
 
         log.debug("Transformer", xml=xml, transformation=transformation, xslt=xslt_path)
 
-        # The Saxon command receives the following parameters:
-        # `-s:-` sets the source to stdin
-        # `-xsl:f{xslt}` sets the xslt to be used, currently located in a file.
-        result = subprocess.run(
-            ["java", "-jar", saxon_path, "-s:-", f"-xsl:{xslt_path}"],
-            stdout=subprocess.PIPE,
-            input=xml,
-        )
+        xslt_proc = self.saxon_processor.new_xslt30_processor()
+
+        node = self.saxon_processor.parse_xml(xml_text=xml.decode("utf-8"))
 
-        # Captured stdout needs to be decoded from bytes to string.
-        return str(result.stdout.decode())
+        result = xslt_proc.transform_to_string(stylesheet_file=xslt_path, xdm_node= node)
```

Comment:
```suggestion
        result = xslt_proc.transform_to_string(stylesheet_file=xslt_path, xdm_node=node)
```
Rather aesthetic: https://www.flake8rules.com/rules/E251.html
![Screenshot from 2021-02-26 09-00-12](https://user-images.githubusercontent.com/763374/109272399-1bbbc600-7811-11eb-9e39-895832f1d4a9.png)